client/state/data-layer/wpcom/sites/gutenberg/index.js

```diff
@@ -29,12 +50,13 @@ export const setType = action =>
 		noop
 	);
 
-const dispatchEditorTypeRequest = dispatchRequestEx( {
+const dispatchEditorTypeSetRequest = dispatchRequestEx( {
 	fetch: setType,
 	onSuccess: noop,
 	onError: noop,
 } );
 
 registerHandlers( 'state/data-layer/wpcom/sites/gutenberg/index.js', {
-	[ EDITOR_TYPE_SET ]: [ dispatchEditorTypeRequest ],
+	[ EDITOR_TYPE_REQUEST ]: [ dispatchSelectedEditorRequest ],
```

Comment:
It took me a while to understand the differences between `EDITOR_TYPE_REQUEST` and `EDITOR_TYPE_SET`, mainly because both handlers have the `Request` suffix and only one of them is using the `Type` part.

I'd use this naming to make it more obvious:

```
{
	[ SELECTED_EDITOR_GET ]: [ dispatchSelectedEditorGetRequest ],
    [ SELECTED_EDITOR_SET ]: [ dispatchSelectedEditorSetRequest ],
}
```

Reply:
If y'all don't mind, I'll do this in a follow up 🙇